hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestBlockManagerSafeMode.java

```diff
@@ -366,6 +368,22 @@ public void testSafeModeMonitor() throws Exception {
     assertFalse(bmSafeMode.isInSafeMode());
   }
 
+  @Test
+  public void testSafemodeRecheckIntervalValue() {
+    Configuration conf = new HdfsConfiguration();
+    long interval = conf.getLong(
+            DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_KEY,
+            DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_DEFAULT);
+
+    assertEquals(interval, DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_DEFAULT);
+
+    conf.setLong(DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_KEY, 2000);
+    interval = conf.getLong(
+            DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_KEY,
+            DFS_NAMENODE_SAFEMODE_RECHECK_INTERVAL_DEFAULT);
+    assertEquals(interval, 2000);
+  }
```

Comment:
What is this test testing? The working of configuration class? That isn't required? What you need to test is the namenode is using the configured value as the recheck interval.